facebook_connector.py

```diff
@@ -126,7 +120,9 @@ def get_cleaned_fully_merged_messages(strip_html_content=True,
     if not _messages_file:
         print("Please initialize the facebook_connector module.")
         return
-    chats = MessageHtmlParser(path=_messages_file).parse()
+    chats = None
+    with io.open(_messages_file, mode="rt", encoding="utf-8") as handle:
```

Comment:
Why this instead of just with open()?  I don't usually see it fully written out.

Also what's mode "rt" instead of "r"?

Reply:
io.open differs from open. It allows specifying the encoding. I tried open, but it just produced garbage. 'rt' is 'read' and 'text mode'.